catalog/content/docs/src/assembly/asciidocs.xml

```diff
@@ -26,11 +26,11 @@
         <fileSet>
              <fileMode>0644</fileMode>
              <directoryMode>0755</directoryMode>
-             <directory>src/main/resources</directory>
+             <directory>src/main/resources/_contents</directory>
```

Comment:
Great question. It's an asciidoctor trick to not render the contents of a directory with that starts with an underscore. The files in those directories are only partials to be included elsewhere, so no need to create a finished version and delete it later.